### PR TITLE
Add a man-page for rpi-imager

### PR DESCRIPTION
**Pull request by waveform80**
Just adds a basic man-page covering the GUI and cli functionality

### CODE REVIEW

File: doc/man/rpi-imager.1 [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+.
+.TP
+.BI \-\-qm \ translations
+Specify an alternate Qt message translations file to use with the GUI.
+.
+.TP
+.BI \-\-repo \ url
+Instead of querying the Raspberry Pi servers for the list of available OS
+images, query this URL instead. The URL will be expected to return a JSON file
+confirming to the format described in the associated
```
Comment (lurch) [inline doc/man/rpi-imager.1:107]: confirming -> conforming ?

File: doc/man/rpi-imager.1 [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Write
+.I test.img
+to
+.I /dev/mmcblk0
+without any verification of the image itself, and no check that the image
+flashed correctly. This also avoids launching the graphical interface.
+.
+.
+.SH SEE ALSO
+.BR dd (1)
```
Comment (lurch) [inline doc/man/rpi-imager.1:158]: Heh :wink:

File: doc/man/rpi-imager.1 (hunk 1 above)
Comment (waveform80) [reply doc/man/rpi-imager.1]: Doh!

File: doc/man/rpi-imager.1 (hunk 2 above)
Comment (waveform80) [reply doc/man/rpi-imager.1:158]: I couldn't resist :)

**Pull request merged**